services/compute/virtualmachine/mockclient.go

```diff
@@ -0,0 +1,434 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache v2.0 License.
+
+package virtualmachine
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/microsoft/moc-sdk-for-go/services/compute"
+	computeavset "github.com/microsoft/moc-sdk-for-go/services/compute/availabilityset"
+	"github.com/microsoft/moc-sdk-for-go/services/network/networkinterface"
+	"github.com/microsoft/moc/pkg/auth"
+	"github.com/microsoft/moc/pkg/errors"
+)
+
+type VirtualMachineMockClient struct {
+	vmClient      VirtualMachineClient
+	avsetInternal computeavset.Service
+}
+
+func NewVirtualMachineMockClient(cloudFQDN string, authorizer auth.Authorizer) (*VirtualMachineMockClient, error) {
```

Comment:
Where is this client used?

Reply:
will remove, this is for akshybrid to test in prrallel.